docs/source/getting_started_guide/creating_a_basic_network.rst

```diff
@@ -0,0 +1,98 @@
++++++++++++++++++++++++++++++++
+Creating a basic 3-node network
++++++++++++++++++++++++++++++++
+
```

Comment:
The preference would be to use the header format as described in https://docs.ansible.com/ansible/latest/dev_guide/style_guide/index.html#header-notation

Since this guide isn't in the index.rst yet, we can't see it on the CI readthedocs preview so I don't know if this different order to the header notations works or not.  This suggestion applies to all files in this guide.